src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -298,6 +303,7 @@
     <Reference Include="System.Collections" />
     <Reference Include="System.Diagnostics.Debug" />
     <Reference Include="System.Diagnostics.Tools" />
+    <Reference Include="System.IO.FileSystem.DriveInfo" />
```

Comment:
@weshaggard / @JeremyKuhne, it's ok for System.IO.FileSystem to depend on System.IO.FileSystem.DriveInfo?

Reply:
that change was reverted with https://github.com/dotnet/corefx/commit/bbda7f35fc46af191978f93434786c218cf5f12c#diff-08292c6147f4cac0cb76f64bf5dd7aa6 as it caused build breaks (https://github.com/dotnet/corefx/issues/31724)

Reply:
Ok, thanks.  Just going through PRs in order from the last few months :)